### ISSUE TITLE
[Accessibility] Pivot TalkBack does not announce tab positions on Android

### ISSUE THREAD

**Issue by lutorre** [microsoft/fluentui-system-icons#294]
**Platform**: Android 11

**Steps to reproduce**:

- Open chrome browser and open a web page that contains a Pivot component with two or more tabs.
- Enable TalkBack 
- Navigate to the first and second tabs within the Pivot component

**Expected result**:

TalkBack announces the tan title followed by the tab position (example, 'Tab one, 1 of two')

**Current Resul**t:

Talkback announces only the tab title

**Comment by lutorre** [comment]
Wrong repository
